src/sagemaker_training/smdataparallel.py

```diff
@@ -33,7 +33,8 @@
 
     # list of exceptions SMDDP wants training toolkit to catch and log
     exception_classes = [x for x in dir(exceptions) if isclass(getattr(exceptions, x))]
-except ImportError:
+# relaxed exception type in case of custom exceptions thrown during import
+except Exception:  # noqa W0703
```

Comment:
does this line help on accepting more types of Exceptions for "import smddp"?
wondering how this could help on those 2 use cases.

Reply:
yes it would help on general import exceptions (library not installed in this docker image) and custom exception classes thrown by SMDDP during import and trying to load custom exceptions (SMDDPConfig error, class not found, etc)